content/en/blog/general/automating-cis-compliance/index.md

```diff
@@ -0,0 +1,427 @@
+
+The open source project: [cis-eks-kyverno](https://github.com/ATIC-Yugandhar/cis-eks-kyverno) implements 62 CIS controls with a multi-tool plan time and runtime validation strategy, comprehensive testing, and automated reporting. This framework showcases cloud native security automation and has been tested extensively with KIND (Kubernetes in Docker) clusters.
+
+## The Compliance Challenge in Cloud Native Environments
+
+The CIS Amazon EKS Benchmark v1.7.0 contains 46 security recommendations across five critical areas:
+
+- **Control Plane Configuration** (Section 2): Audit logging, endpoint security - 2 controls
+- **Worker Node Security** (Section 3): Kubelet configuration, file permissions - 13 controls  
+- **RBAC & Service Accounts** (Section 4): Access controls, service account security - 15 controls
+- **Pod Security Standards** (Section 5): Container runtime security - 9 controls
+- **Managed Services** (Section 5): ECR, networking, encryption - 7 additional controls
+
+Traditional compliance approaches typically involve:
+
+1. Manual kubectl commands for configuration checks
+2. SSH access to worker nodes for file permission audits
+3. Periodic reviews of infrastructure configurations
+4. Manual report generation for security assessments
+5. Reactive remediation after issues are discovered
+
+### Challenges with manual approaches:
+
+- **Scale limitations**: Manual processes don't scale across multiple Kubernetes clusters
+- **Time-intensive**: Comprehensive audits can take days per cluster
+- **Error-prone**: Human oversight can miss critical security configurations
+- **Reactive nature**: Issues discovered weeks after deployment
+- **Inconsistency**: Different teams may interpret controls differently
+- **Cloud native complexity**: Traditional tools don't understand Kubernetes-native resources
+
+The cloud native opportunity: Modern CNCF ecosystem tooling offers the potential to automate most compliance checks, shift security left in the development process and provide continuous monitoring. The key is leveraging CNCF technologies that work well together and understand cloud native patterns.
+
+## The Multi-Tool Cloud Native Approach to Complete CIS Coverage
+
+Modern cloud native compliance requires acknowledging that no single tool can validate all CIS controls due to the diverse nature of the requirements. This implementation combines the strengths of multiple CNCF and cloud native security tools:
+
+![Architecture Diagram](./cis-exec.png)
+
+- **Kyverno** (CNCF Incubating): Kubernetes-native policy engine for runtime validation
+- **kube-bench**: Community-standard CIS compliance scanner for node-level validation
+- **OpenTofu**: Open-source infrastructure-as-code for plan-time validation
```

Comment:
This makes it seem like OpenTofu is performing validations. Should we clarify that its using Kyverno for plan time validations?